oeps/processes/oep-0056-architectural-process.rst

```diff
@@ -0,0 +1,191 @@
+***********************************************
+
+For any scope of decision, the decision maker's responsibility is to record the advice they received, whether or not they chose to take it, and why.  This is, certainly, a time investment, but we believe it will both lead to better
+
+Decisions are an opportunity to learn and democratize knowledge
+***************************************************************
+
+Assume decisions will be made
+*****************************
+
+As a rule of thumb, we should open up an ADR pull request in an appropriate repository when starting a new project.  This may not be the only ADR we create, but it should serve as a reminder and a place to take notes as we start to make decisions.
+
+We want ADRs to be a central component of our architectural process.  It's alright for them to evolve (and perhaps split) as we discover the shape of an effort.
+
+
+Approachable Decision Making
+----------------------------
+
+The process of making a technical decision is not atomic, nor do many parts of it occur while writing a document about the decision.  Many decisions are *made* while writing code as we problem solve and navigate the idiosyncracies of the system.  The goal of our process should be to find a way of capturing those decisions
+
+Decision Maker's Checklist
+--------------------------
+
+In order to help ADR authors ensure they get advice from the proper set of stakeholders, we will create a checklist of possible groups/individuals to choose from.  It's the responsibility of the author to ensure the relevant people are aware of the ADR and are given the opportunity to provide feedback, either individually, in separate sessions, or in the advisory forum.
+
+Technical Strategic Review
+--------------------------
+
+Coinciding with the broader product strategy and prioritization cadence, establish quarterly strategic reviews focused on engineering and architectural priorities as they relate to our business and platform needs.  There may several reviews, and they may be domain/theme-specific depending on the organization.
+
+The review should be attended by engineering managers and at least one technical lead from each team in the
+
+
+
+Review ADRs at the Architectural Advisory Forum
+-----------------------------------------------
+
+
+Organize quarterly engineering strategic review sessions
+--------------------------------------------------------
+
+
+Options Considered
+==================
+
+Decentralize Decision Making and Promote ADRs
+---------------------------------------------
+
+This is not the first option we arrived at, but feels as if it most directly addresses the scalability and empowerment issues we're seeing in our process.
+
+Invite and Involve Senior Individual Contributors (ICs) to Product Rituals
+--------------------------------------------------------------------------
+
+What if we just invite engineers to Strategic Review and Consumer Review so they can get involved earlier?
```

Comment:
what is Strategic Review and Consumer Review? :)

Reply:
Yeah, I'm trying to remove references to those names. 😄  Those are 2U-specific ceremonies; trying to use more generic language.  That's left over from a prior iteration of this document that was 2U-specific.